src/utils/qa/qa.js

```diff
@@ -0,0 +1,12 @@
+const doc = document;
+
+/**
+ * @function qa
+ * @desc query selector all with results as array
+ * @param {string} [el] element to search for
+ * @param {node} context element or document to search inside of
+ * @return {array} - Elements that matched query.
+ */
+const qa = (el, context = doc) => [].slice.call(context.querySelectorAll(el));
+
+export default qa;
```

Comment:
I'm personally a fan of verbose naming and might be more consistent with the more verbose naming of `createEl` but I'm not overly fussed.